climt/_lib/GFS/Makefile

```diff
@@ -1,4 +1,13 @@
-include Makefile.conf
+OMPFLAG = -fopenmp
```

Comment:
I assume all of this is correct. You may want to make the way you're doing spaces around the equal signs uniform.